docs/changelog.md

```diff
@@ -66,7 +66,7 @@ description: |
   - Fixed empty pages appearing when a [context] expression wraps whole pages
   - Fixed `{set block(spacing: x)}` behaving differently from
     `{set block(above: x, below: x)}`
-  - Fixed behaviour of [`rotate`] and [`scale`] with `{reflow: true}`
+  - Fixed behavior of [`rotate`] and [`scale`] with `{reflow: true}`
```

Comment:
Just my 2 cents: I would have kept the British english way of writing by default.

Reply:
The codebase uses primarily American English, e.g. `color`. I personally use a mix... I also prefer "behaviour", but a bit of consistency is okay.

Reply:
Fair enough ;)